CODE_OF_CONDUCT.md

```
@@ -1,23 +1,128 @@
+standards, including sustained inappropriate behavior, harassment of an
+individual, or aggression toward or disparagement of classes of individuals.
+
+**Consequence**: A permanent ban from any sort of public interaction within
+the community.
+
+## Attribution
 
-## Contact
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)